### ISSUE TITLE
M.2 Wireless Buyers Guide - "Fenvi AC1200" is not an M.2 card

### ISSUE THREAD

**Issue by bootlegseahorse** [dortania/bugtracker#305]
First I want to express my appreciation to Dortania and Acidanthera for the efforts to make us boot macOS on Legacy Macs and Hackintoshes.

Today whiles checking [the M.2 section in the Wireless Buyers Guide](https://dortania.github.io/Wireless-Buyers-Guide/types-of-wireless-card/m2.html#supported-cards), I noticed a possible issue. Under "Supported Cards - BCM94352Z" section, the [Fenvi AC1200](https://www.fenvi.cn/product_detail_24.html) wireless card is listed. But according to the Fenvi official website, the Fenvi AC1200 is actually a PCIe card (See: [FV-AC1200 - Product Details](https://www.fenvi.cn/product_detail_24.html)). 

There is a A+E M.2 chip based on BCM94352Z manufactured by Fenvi, though, but its product name is not AC1200. It's simply called [Fenvi BCM94352Z](https://www.fenvi.cn/product_detail_40.html). And according to some buyer feedbacks on AliExpress, this Fenvi chip is fully supported on macOS up to Ventura. 

So I think it might be necessary to replace [Fenvi AC1200](https://www.fenvi.cn/product_detail_24.html) with [Fenvi BCM94352Z](https://www.fenvi.cn/product_detail_40.html) on the M.2 Wireless Buyers Guide page.

Hope this helps

**Comment by bootlegseahorse** [comment]
Update: I've just tried the [Fenvi BCM94352Z](https://www.fenvi.cn/product_detail_40.html) on macOS Ventura and I can confirm that it's fully supported, even without the Brcm fixup kexts.

**Comment by JamisonMurphy** [comment]
> Update: I've just tried the [Fenvi BCM94352Z](https://www.fenvi.cn/product_detail_40.html) on macOS Ventura and I can confirm that it's fully supported, even without the Brcm fixup kexts.

@bootlegseahorse 
I plan to buy one too. Does Airdrop work both direction in macos ventura?
Also ,need kext for it?

**Comment by bootlegseahorse** [comment]
> 



> > Update: I've just tried the [Fenvi BCM94352Z](https://www.fenvi.cn/product_detail_40.html) on macOS Ventura and I can confirm that it's fully supported, even without the Brcm fixup kexts.
> 
> @bootlegseahorse I plan to buy one too. Does Airdrop work both direction in macos ventura? Also ,need kext for it?

Yup. All perfectly functional - AirDrop, Handoff, Continuity Camera, AirPlay, Universal Control, etc. Without any kexts needed.

I bought one really cheap; the only problem is that the card's WiFi speed is limited to 434 Mbps. Anyway it's enough for me - not sure if that'll satisfy your need though.

**Comment by JamisonMurphy** [comment]
My fenvi 94352z wifi not working in Macos13.4 ,but bluetooth work.

**Comment by bootlegseahorse** [comment]
> My fenvi 94352z wifi not working in Macos13.4 ,but bluetooth work.

You're probably using a broken card, or loading broken kexts. Mine works perfectly with macOS 13.4.1 (though limited to 434 Mbps). 

Anyway, you could try these methods:

- Sleep and wake from sleep
- Delete all WiFi-related kexts
- Reboot your Mac
- Reboot to Windows 10/11 and see if WiFi works
- Swap the card's antennas
- Open Hackintool and see if the card is loaded successfully

**Comment by JamisonMurphy** [comment]
I tried many ways (included ways you proposed), wifi and BT work fine in win11,but in Hackintool only in PCIe part show this wifi card(bcm 4360), I suspect this card is broken or my computer compatibility issue. So I have returned the product.

**Comment by JamisonMurphy** [comment]
Do you have serious issues with Bluetooth and WiFi 2.4g interference ?(I have) 
Because the Bluetooth and WiFi of Fenvi 94352z use the same antenna.

**Comment by bootlegseahorse** [comment]
> Do you have serious issues with Bluetooth and WiFi 2.4g interference ?(I have) Because the Bluetooth and WiFi of Fenvi 94352z use the same antenna.

I'm not really sure about that, because I almost always use 5GHz WiFi. Never noticed the Bluetooth interference. 

By the way, nice to know that the WiFi now works with your computer..
